websites/N/Notespace/presence.ts

```diff
@@ -0,0 +1,88 @@
+const presence = new Presence({
+	clientId: "1193651901221306378",
+});
+
+const enum Assets {
+	Icon = "https://i.imgur.com/W7Ybot8.png",
+}
+
+presence.on("UpdateData", async () => {
+	const currentPath = document.location.pathname,
+		presenceData: PresenceData = {
+			details: "Twoja wiedza jest tu",
+			largeImageKey: Assets.Icon,
+			smallImageKey: Assets.Viewing,
+			smallImageText: "Przegląda...",
+			startTimestamp: new Date(),
```

Comment:
Devine the startTimestamp outside your update data, as browsingStamp or something like that